pyproject.toml

```diff
@@ -93,6 +93,12 @@ cupy-cuda12x = "^12.0.0"
 cuquantum-python-cu12 = "^23.3.0"
 qibojit = { git = "https://github.com/qiboteam/qibojit.git" }
 
+[tool.poetry.group.cloud]
+optional = true
+
+[tool.poetry.group.cloud.dependencies]
+qibo-cloud-backends = { git = "https://github.com/qiboteam/qibo-cloud-backends.git" }
+
```

Comment:
Instead of group (that is a Poetry feature, not passing to the package) you should use a pip extra, let's say cloud, such that it will be possible to install with:
```sh
pip install qibo[cloud]
```
https://python-poetry.org/docs/pyproject/#extras

In practice, just move the dependency up, and mark it with `optional = true`. Then, add the extra group, (here it has been never done, and even the other groups are mostly abusing this Poetry feature, i.e. all the cuda groups...).
If you want to have an example you could also have a look to Qibolab and Qibocal.

Reply:
Solved in 1d12b45